src/orderbook.ts

```diff
@@ -293,6 +301,14 @@ export class Orderbook {
   }
 }
 
+export interface OrderbookJson {
+  baseToken: string;
+  quoteToken: string;
+  remainingFractionAfterFee: Fraction | FractionJson;
+  asks: Record<string, Offer | OfferJson>;
+  bids: Record<string, Offer | OfferJson>;
```

Comment:
Why allow this flexibility? If it comes from json then it should always be `OfferJson`, right?

Reply:
Its because `toJSON` method doesn't convert `Offer` to `OfferJson`.

Reply:
Hmm... I guess I can create 2 types here instead of just one. That might be a more correct approach.

Reply:
@e00E I split it into 2 types now. Let me know if you think this is better.